runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -126,8 +128,12 @@ public CheckpointWriter(CorfuRuntime rt, UUID streamId, String author, T map) {
      */
     public Token appendCheckpoint() {
         long start = System.currentTimeMillis();
+        //TODO:Add comments.
+        long address = rt.getStreamsView().append(new MultiSMREntry(), null, streamId);
```

Comment:
Complete this comment. Just mention how this adds a no-op to the stream while maintaining the backpointers (i.e. different than a hole).